README.md

```diff
@@ -8,22 +8,31 @@ In order to run this demo, you will need a Dolby.io account. If you don't alread
 
 ## Features and roadmap
 
-- [x] Publish and view streams
-- [x] Invite viewers to watch your stream
-- [x] Mute audio and video both as publisher and viewer
-- [x] Screen sharing
+### RTS Publisher
+
+- [x] Publish streams
 - [x] Publish with simulcast (only on Chrome)
 - [x] Microphone and camera device selection
+- [x] Invite viewers to watch your stream
+- [x] Screen sharing
+- [x] Mute audio and video
 - [x] Resolution, bandwidth, codec, and bitrate selection
-- [x] Observing stream statistics
-- [ ] Ability to stream from two or more sources
+- [x] Ability to stream from two or more sources
```

Comment:
We can only stream 4 at most, so I suggest change it to
```suggestion
- [x] Ability to stream up to 4 sources
```

Reply:
Thank you, fixed.